_11ty/img-perf.js

```diff
@@ -1,29 +1,26 @@
-const { JSDOM } = require("jsdom");
+const { parseHTML } = require('linkedom')
```

Comment:
5️⃣  Rewriting this transform to use `linkedom` makes it _way_ faster. On my machine, it cuts build time by ~50%, with average times dropping from 60ms to 30ms.

As far as I can tell, the transform doesn't actually need any async functionality so I removed the async decorators, as well.